src/fake_keyboard.cpp

```diff
@@ -325,10 +329,62 @@ void fake_keyboard::ydo_type_char(uint32_t c) {
     }
 }
 
+// Set the clipboard text
+QString fake_keyboard::copy_to_clipboard(const QString &text) {
+    if (text.isEmpty()) {
+        LOGE("Text is empty, skipping clipboard copy");
+        return QString("");
```

Comment:
```suggestion
        return {};
```

To be consistent with the style used in this project.